### PR TITLE
Remove `GETU32`, `PUTU32`, `BSWAP4`, `BSWAP8` macros. 

### PR DESCRIPTION
**Pull request by frewsxcv**
I agree to license my contributions to each file under the same terms
given at the top of each file I changed.

### CODE REVIEW

**Comment by briansmith** [comment]
Your change is fine but I think it should be squashed woth the upcoming changes to crypto/modes/internal.h and crypto/modes/gcm.c.

**Comment by frewsxcv** [comment]
Okay. I'll just update this PR with those changes once I'm done modifying those files

**Comment by frewsxcv** [comment]
This is ready for review now

File: crypto/internal.h [hunk 1]
```diff
@@ -501,6 +501,23 @@ static inline uint32_t from_be_u32_ptr(const uint8_t *data) {
 #endif
 }
 
+
+/* to_be_u32_ptr transforms 32-bit value |x| to big-endian-encoded and sets it
+ * at |data| */
+static inline void to_be_u32_ptr(uint8_t *data, uint32_t x) {
```
Comment (frewsxcv) [inline crypto/internal.h]: Alternatively, I could have this just return a big-endian-encoded `uint32_t` and each caller is responsible for inserting it into `data`

**Comment by frewsxcv** [comment]
Comments have been addressed

**Comment by frewsxcv** [comment]
This closes #69

**Comment by frewsxcv** [comment]
This also closes #94 ?

**Comment by briansmith** [comment]
Thanks! I removed the `is_endian` definitions that were causing problems for Windows builds and landed this as c882c2c3ec837f14267991122dc20e6aba5700bc.

**Pull request closed without merging**